### PR TITLE
[skip-ci][DOC] Clarify how we want to be cited in README.md

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,8 @@ acquisition, simulation and data analysis systems.
 | v6-18-00-patches | [![Build Status](http://lcgapp-services.cern.ch/root-jenkins/buildStatus/icon?job=root-nightly-v6-18-00-patches)](https://lcgapp-services.cern.ch/root-jenkins/view/ROOT/job/root-nightly-v6-18-00-patches/) |
 
 ## Cite
-We are [![DOI](https://zenodo.org/badge/10994345.svg)](https://zenodo.org/badge/latestdoi/10994345)
 
-Please cite us as
+When citing ROOT, please use both the reference reported below and the the DOI specific to your ROOT version available [on Zenodo](https://zenodo.org/badge/latestdoi/10994345). For example, you can copy-paste and fill in the following citation:
```
Comment (hageboeck) [inline README.md:41]:
```suggestion
When citing ROOT, please use both the reference reported below and the DOI specific to your ROOT version available [on Zenodo](https://zenodo.org/badge/latestdoi/10994345). For example, you can copy-paste and fill in the following citation:
```

**Comment by hageboeck** [comment]
However, Zenodo seems to be messing up the versions:
https://zenodo.org/record/3895860

That's where I land, and it says 6.18 and 6.20 on the same page.

**Comment by eguiraud** [comment]
@Axel-Naumann our zenodo things are broken: clicking on v6.20/06 a banner is shown that says there is a newer version, and if I click on that it brings me to 6.18/02 (which, in the text, claims to be a 6.20 patch release)

**Comment by Axel-Naumann** [comment]
> clicking on v6.20/06 a banner is shown that says there is a newer version,

I contacted zenodo on that. 

> and if I click on that it brings me to 6.18/02 (which, in the text, claims to be a 6.20 patch release)

I fixed that, thanks!

File: README.md [hunk 2]
```diff
@@ -37,9 +37,8 @@ acquisition, simulation and data analysis systems.
 | v6-18-00-patches | [![Build Status](http://lcgapp-services.cern.ch/root-jenkins/buildStatus/icon?job=root-nightly-v6-18-00-patches)](https://lcgapp-services.cern.ch/root-jenkins/view/ROOT/job/root-nightly-v6-18-00-patches/) |
 
 ## Cite
-We are [![DOI](https://zenodo.org/badge/10994345.svg)](https://zenodo.org/badge/latestdoi/10994345)
```
Comment (Axel-Naumann) [inline README.md:40]: Can we keep the SVG? It's pretty common.

File: README.md [hunk 3]
```diff
@@ -37,9 +37,8 @@ acquisition, simulation and data analysis systems.
...
 
-Please cite us as
+When citing ROOT, please use both the reference reported below and the DOI specific to your ROOT version available [on Zenodo](https://zenodo.org/badge/latestdoi/10994345). For example, you can copy-paste and fill in the following citation:
```
Comment (Axel-Naumann) [inline README.md:41]:
```suggestion
When citing ROOT, please use both the reference reported below and the DOI specific to your ROOT version available [on Zenodo](https://zenodo.org/badge/latestdoi/10994345) [![DOI](https://zenodo.org/badge/10994345.svg)](https://zenodo.org/badge/latestdoi/10994345). For example, you can copy-paste and fill in the following citation:
```

**Pull request merged**